tests/testScripts/test_landing.py

```diff
@@ -1,7 +1,10 @@
 # It'll be the importing of report tools here
 
 import config
+import time
 import pytest
+from src.resources.config_credentials import irina_email, irina_password, irina_account_name, wrong_email, \
```

Comment:
You can simplify the whole credential process with a different approach.

Reply:
Done. Added credentials to config.py